assembl/static2/js/app/components/debate/common/postTranslate.jsx

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import { Translate, I18n } from 'react-redux-i18n';
+import SwitchButton from 'react-switch-button';
+import 'react-switch-button/dist/react-switch-button.css';
```

Comment:
Doesn't adding this via webpack put the entire CSS of the switch button in the root of the `bundle.js` file, increasing the html file size and not leveraging the async fetch of stylesheets by the browser?

Reply:
For production, there is a separate bundle.css.